src/js/fetching/fetch-movies.js

```diff
@@ -79,15 +74,17 @@ export const getMoviesByTitle = async movieTitle => {
     }
     console.log(`Poniżej przykladowy console.log dla filmu "${movieTitle}"`);
     console.log(data);
-    loadMovie();
-    //TO DO function here!
+    showLoader();
+
+    //TODO function here!
     renderMovies(data.results);
   } catch (error) {
     console.error(error);
   }
 };
 
-//  4.    ------ Function - fetch - get movie details object by movie ID ------
+//  4.    ------ Function fetch - get movie (details object) by movie ID ------
+
```

Comment:
The comment must be on the line above the content being described (no break line)